edb/lib/std/70-converters.edgeql

```diff
@@ -238,24 +238,24 @@ std::to_str(json: std::json, fmt: OPTIONAL str={}) -> std::str
 
 
 CREATE FUNCTION
-std::to_str(array: array<std::str>, delimiter: std::str) -> std::str
+std::array_join(array: array<std::str>, delimiter: std::str) -> std::str
 {
     SET volatility := 'STABLE';
     USING SQL $$
-    SELECT array_to_string("array", "delimiter");
+    SELECT array_array_join("array", "delimiter");
     $$;
 };
 
 
 CREATE FUNCTION
-std::to_array(s: std::str, delimiter: std::str) -> array<std::str>
+std::str_split(s: std::str, delimiter: std::str) -> array<std::str>
```

Comment:
This function should be moved to https://github.com/edgedb/edgedb/blob/master/edb/lib/std/30-strfuncs.edgeql